src/averageAges.js

```diff
@@ -14,12 +14,22 @@
  *
  * @return {number}
  */
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const selection = people.filter(
+    man => {
```

Comment:
- should have parentheses
```suggestion
    (man) => {
```

Reply:
- we don't know the sex of the person we're currently iterating

Reply:
👌fixed
parentheses added and it's a  `person` now